<launch>
	<node name="smach_viewer" pkg="smach_viewer" type="smach_viewer.py"/>
	<node name="data_captation" pkg="data_capture" type="data_captation.py"/>
	<node name="operating_mode" pkg="data_capture" type="operating_mode.py" output="screen"/>

	<arg name="battery_runtime" default="300"/>

	<node pkg="data_capture" name="battery_simulator" type="battery_simulator.py" output="screen" clear_params="true">
		<param name="rate" value="5" />
		<param name="battery_runtime" value="$(arg battery_runtime)" />
	</node>
</launch>